app/manifest.json

```diff
@@ -52,6 +52,192 @@
         "file://*/*.txt.*"
       ],
       "js":[
+        "js/vendor/highlight.js/highlight.min.js",
```

Comment:
My initial concern was to load every single language but using a client-side highlighting is not reliable...